osu.Game/Overlays/Rankings/Tables/PerformanceTable.cs

```diff
@@ -24,7 +24,7 @@ public PerformanceTable(int page, IReadOnlyList<UserStatistics> rankings)
 
         protected override Drawable[] CreateUniqueContent(UserStatistics item) => new Drawable[]
         {
-            new RowText { Text = item.PP.ToLocalisableString(@"N0"), }
+            new RowText { Text = item.PP?.ToLocalisableString(@"N0"), }
```

Comment:
I'm unsure about this one, the [osu-web API documentation](https://osu.ppy.sh/docs/index.html#userstatistics) state that `UserStatistics.PP` is not nullable, but for some reason it's specified as `decimal?` in that class.

I've used null conditional as a safe way to handle it. An inspection on this line will pop up when `PP` becomes non-null anyways.